doc/source/conf.py

```diff
@@ -301,25 +301,38 @@
 }
 
 
-common_imports = """\
+header = """\
 .. currentmodule:: pandas
 
 .. ipython:: python
    :suppress:
 
+   import csv
+   from datetime import datetime, timedelta
+   import functools
+   import glob
+   import itertools
+   import os
+
    import numpy as np
    from pandas import *
```

Comment:
This `from pandas import *` should also not be here. If it was needed for the whatsnew files (where the common imports were used now?), I would still keep that separate until we fix the examples in the whatsnew files to not need this.

Reply:
> This `from pandas import *` should also not be here. If it was needed for the whatsnew files (where the common imports were used now?), I would still keep that separate until we fix the examples in the whatsnew files to not need this.

Okay - I have added `from pandas import *` to all whatsnew files. We have to revisit all the whatnew files.

Reply:
I have added `from pandas import * ` to all the whatsnew file. We have to review all of them later to see where files actually need the import (my guess is all of them).